PrivateCloud.DiagnosticInfo/PrivateCloud.DiagnosticInfo.psm1

```diff
@@ -5086,16 +5084,72 @@ WARNING: the SMB Client is receiving RDMA disconnects. This is an error whose ro
 `t cluster node reboots are a natural & expected source of disconnects.
 "@
 
-    $j += Start-Job -name 'SMB Connectivity Error Check - Disconnect Failures (Event 30804)' -InitializationScript $CommonFunc -ScriptBlock $ReportTableBlock -ArgumentList $eventlogs,30804,$CaptureDate,([ConsoleColor]'Red'),$w
+    $j += Start-Job -name 'SMB Connectivity Error Check - RDMA Disconnect Failures (Event 30804)' -InitializationScript $CommonFunc -ScriptBlock $ReportTableBlock -ArgumentList $eventlogs,30804,$CaptureDate,([ConsoleColor]'Red'),$w,@{'ConnectionType'='=2'}
 
     $w = @"
 WARNING: the SMB Client is receiving RDMA connect errors. This is an error whose root
 `t cause may be actual lack of connectivity or fundamental problems with the RDMA
 `t network fabric. Please inspect especially if in the Last5 bucket.
 "@
 
-    $j += Start-Job -name 'SMB Connectivity Error Check - Connect Failures (Event 30803)' -InitializationScript $CommonFunc -ScriptBlock $ReportTableBlock -ArgumentList $eventlogs,30803,$CaptureDate,([ConsoleColor]'Yellow'),$w
+    $j += Start-Job -name 'SMB Connectivity Error Check - RDMA Connect Failures (Event 30803)' -InitializationScript $CommonFunc -ScriptBlock $ReportTableBlock -ArgumentList $eventlogs,30803,$CaptureDate,([ConsoleColor]'Yellow'),$w,@{'ConnectionType'='=2'}
+
+    $w = @"
+
+WARNING: the SMB Client is receiving TCP/IP disconnects. Note that
+
+`t cluster node reboots are a natural & expected source of disconnects.
+
+"@
+
+    $j += Start-Job -name 'SMB Connectivity Error Check -  TCP/IP Connect Failures (Event 30804)' -InitializationScript $CommonFunc -ScriptBlock $ReportTableBlock -ArgumentList $eventlogs,30804,$CaptureDate,([ConsoleColor]'Yellow'),$w, @{'ConnectionType'='=1'}
+
+    $w = @"
+
+WARNING: the SMB Client is receiving TCP/IP connect errors. 
+
+Please inspect especially if in the Last5 bucket.
+
+"@
+
+    $j += Start-Job -name 'SMB Connectivity Error Check - TCP/IP Connect Failures (Event 30803)' -InitializationScript $CommonFunc -ScriptBlock $ReportTableBlock -ArgumentList $eventlogs,30803,$CaptureDate,([ConsoleColor]'Yellow'),$w, @{'ConnectionType'='=1'}
+
+    $w = @"
+
+WARNING: the SMB Client is  experiencing long time to complete IO.
+
+SMB will close diconnected channels

```

Comment:
"disconnected"

Will each of these show up as disconnected 30804? What's the relationship?